diff --git a/twrpApex.cpp b/twrpApex.cpp
index 557283b..a461a9f 100644
--- a/twrpApex.cpp
+++ b/twrpApex.cpp
@@ -12,7 +12,6 @@ bool twrpApex::loadApexImages() {
 	char* additionalApexFiles = std::strtok(additionalFiles, " ");
 #endif
 
-	apexFiles.push_back(APEX_DIR "/com.android.apex.cts.shim.apex");
 	apexFiles.push_back(APEX_DIR "/com.google.android.tzdata2.apex");
 	apexFiles.push_back(APEX_DIR "/com.android.tzdata.apex");
 	apexFiles.push_back(APEX_DIR "/com.android.art.release.apex");
@@ -116,7 +115,9 @@ bool twrpApex::mountApexOnLoopbackDevices(std::vector<std::string> apexFiles) {
 		}
 		bool load_result = loadApexImage(fileToMount, device_no);
 		if (!load_result) {
-			return false;
+			LOGINFO("Skipping apex that failed to load: %s\n", fileToMount.c_str());
+			device_no++;
+			continue;
 		}
 		device_no++;
 	}
-- 
2.55.0

