diff --git a/fs_mgr/libfs_avb/util.cpp b/fs_mgr/libfs_avb/util.cpp
index 7783d04e68..d57ac5f755 100644
--- a/fs_mgr/libfs_avb/util.cpp
+++ b/fs_mgr/libfs_avb/util.cpp
@@ -110,7 +110,7 @@ bool IsDeviceUnlocked() {
     std::string verified_boot_state;
 
     if (fs_mgr_get_boot_config("verifiedbootstate", &verified_boot_state)) {
-        return verified_boot_state == "orange";
+        return (verified_boot_state == "orange" || verified_boot_state == "green");
     }
     return false;
 }

